containerd.service

```diff
@@ -32,7 +32,7 @@ RestartSec=5
 # in the kernel. We recommend using cgroups to do container-local accounting.
 LimitNPROC=infinity
 LimitCORE=infinity
-LimitNOFILE=infinity
+LimitNOFILE=1048576
```

Comment:
```suggestion
```

For the equivalent `docker.service` config, the [decision was to drop this line](https://github.com/moby/moby/pull/45534#pullrequestreview-1543977154), and it'll be part of the v25 release of moby.

Equivalent change should hopefully follow soon here 🙏

Reply:
No worries, I know how it is :)

Thanks for kickstarting the process with the original attempt here! ❤️